src/java/org/apache/cassandra/streaming/StreamCoordinator.java

```diff
@@ -42,17 +42,19 @@
     private final Map<InetAddressAndPort, HostStreamingData> peerSessions = new HashMap<>();
     private final StreamOperation streamOperation;
     private final int connectionsPerHost;
+    private final boolean initiator;
     private StreamConnectionFactory factory;
     private Iterator<StreamSession> sessionsToConnect = null;
     private final UUID pendingRepair;
     private final PreviewKind previewKind;
 
     public StreamCoordinator(StreamOperation streamOperation, int connectionsPerHost, StreamConnectionFactory factory,
-                             boolean connectSequentially, UUID pendingRepair, PreviewKind previewKind)
+                             boolean initiator, boolean connectSequentially, UUID pendingRepair, PreviewKind previewKind)
```

Comment:
For consistency with `StreamSession`, what about passing "follower", rather than "initiator"?

Reply:
good idea, updated.